main.css

```diff
@@ -821,3 +853,12 @@ a:hover .profile__portrait {
   border-radius:         4px;
 }
 
+.tutorial__steps h4,
+.tutorial__steps p,
+.tutorial__steps code {
+  margin: 0.6em 0;
+}
+
+.tutorial__steps li {
+  margin: 2em 0;
+}
```

Comment:
better layout/spacing for tutorial

before:
![bildschirmfoto 2016-01-19 um 10 35 04](https://cloud.githubusercontent.com/assets/37363/12416977/23968e6c-bea5-11e5-9652-dd80fd6a7126.png)

after:
![bildschirmfoto 2016-01-19 um 10 35 04](https://cloud.githubusercontent.com/assets/37363/12416981/28197c42-bea5-11e5-96d2-cb959af04a88.png)